### ISSUE TITLE
devcontainer CLI no longer supports "open" option

### ISSUE THREAD

**Issue by ManfredLange** [microsoft/vscode-docs#5041]
# Desired State
The documentation is out of date and should be updated accordingly.

# Actual State

According to the documentation at https://code.visualstudio.com/docs/remote/devcontainer-cli the output of `devcontainer --help` lists a command named `open`:
```bash
$ devcontainer --help
devcontainer <command>

Commands:
  devcontainer open [path]   Open a dev container in VS Code
  devcontainer build [path]  Build a dev container image

Options:
  -h, --help               Show help  [boolean]
      --disable-telemetry  Disable telemetry  [boolean] [default: false]
```

However, a few days ago and after following the instructions to install it on Ubuntu, the output of `devcontainer --help` looks as follows:
```bash
manfred@DESKTOP-RKN4UE3:~/projects/optarix/backend$ devcontainer --help
devcontainer <command>

Commands:
  devcontainer build [path]  Build a dev container image

Options:
  -h, --help               Show help                                                                           [boolean]
      --disable-telemetry  Disable telemetry                                                  [boolean] [default: false]
```

As you can see it list only `build` as a command. `open` is no longer available.

**Comment by gregvanl** [comment]
Hi @ManfredLange I can't reproduce this on either Windows or Ubuntu.

![image](https://user-images.githubusercontent.com/12818376/146663308-1dd9344f-c650-46f2-9a48-05f8b25cfdcb.png)

I installed the most recent version of the Remote - Containers extension (v0.209.6) and ran the Remote - Containers: Install devcontainer CLI command.

Could you have an older version of the Remote - Containers extension installed or maybe you have an older version of the devcontainer CLI on your Ubuntu $PATH?

That documentation was just reviewed and updated on 12/17/2021 so I'd be surprised if it was out of date.

**Comment by ManfredLange** [comment]
@gregvanl  Thank you for your quick reply, Greg!

I already have version 0.209.6 installed. Still no luck. The installation script of "Remote - Containers: Install devcontainer CLI" command kept telling me that it was in the PATH. However, both, bash and Powershell terminals, kept telling me that the command `devcontainer` doesn't exist.

Turns out that restarting WSL and therefore the distro (Ubuntu 20.04 in my case), and restarting Windows Terminal wasn't sufficient.

Restarting the computer, i.e. the Windows host, resolved the issue in my case. It appears as if certain changes will only be applied then but I don't know the specifics. In my case there must have been some residual changes that made it only after a reboot of the Windows host (good old Windows: give it a proper reboot once in a while to keep it happy ...).

Long story short: If you install the devcontainer CLI it pays to reboot the Windows host. 

Also, if you want to check what the PATH looks like after the installation, look for an entry like the following:
```bash
/mnt/c/Users/<username>/AppData/Roaming/Code/User/globalStorage/ms-vscode-remote.remote-containers/cli-bin
```
where you need to replace `<username>` with your Windows user name.

**Comment by allenporter** [comment]
Hi, I followed the instructions on the website for installing outside of vscode and saw this today:

```
$ npm install -g @vscode/dev-container-cli
/home/allen/.nvm/versions/node/v14.18.2/bin/devcontainer -> /home/allen/.nvm/versions/node/v14.18.2/lib/node_modules/@vscode/dev-container-cli/cli.js
+ @vscode/dev-container-cli@0.209.6
$ devcontainer --help
devcontainer <command>

Commands:
  devcontainer build [path]  Build a dev container image

Options:
  -h, --help               Show help                                                       [boolean]
      --disable-telemetry  Disable telemetry                              [boolean] [default: false]
```

Only shows `build` without an `open`.
